### ISSUE TITLE
Update dev branch, build, and related to create 2 VSIXes for Dev14 and Dev15

### ISSUE THREAD

**Issue by deepakaravindr** [NuGet/Home#2386]

**Comment by emgarten** [comment]
@deepakaravindr is this complete?

**Comment by deepakaravindr** [comment]
@emgarten. Not fully.

dev branch already has the changes necessary to vsixmanifest file to build Dev14 just like the release branch.

However, we have not fixed the code fully to be able to build dev14 and dev15 VSIX out of the same branch. I would say that this tracking issue is depended on the following issues assigned to @drewgil 

https://github.com/NuGet/Home/issues/2392
https://github.com/NuGet/Home/issues/2369

At this point, this is merely a tracking issue for the build changes that needs to happen once the issues above are fixed

**Comment by yishaigalatzer** [comment]
@deepakaravindr is this completely done?

**Comment by deepakaravindr** [comment]
@yishaigalatzer, No. It is blocked on the dependency bugs

**Comment by yishaigalatzer** [comment]
I don't quite follow - Can we produce dev14 and dev15 builds for 3.4.2 (which is what I think this bug is tracking for 3.4.2)?

**Comment by drewgil** [comment]
We only have one build at this point, which works fine against Dev14 and *mostly* fine against Dev15 (except for the cpp install and TFS issues described above). We do not have the ability to build a pure Dev15 VSIX yet, which is the work Deepak is referring to.

**Comment by deepakaravindr** [comment]
@yishaigalatzer  and @drewgil.
Yes, we can build 3.4.2 for Dev14 and Dev15 from different branches. Just not off a single branch.
If we preferred, we can close this as dupe of https://github.com/NuGet/Home/issues/2293.

**Comment by rrelyea** [comment]
@drewgil - you've addressed this already, right?

**Comment by drewgillies** [comment]
The quick answer is Yes. Remaining work includes the following, and should be tracked in separate issues:
- ```
     Lighting up tests, the Dev14 component of which is already tracked (a sxs Dev14/15 install broke tests prior to any of the Dev15 work, and this is still to be addressed)
  ```
- ```
     I may yet file an issue for Dev15 tests if there is separate work there after fixing the Dev14 tests
  ```
- ```
     CI integration
  ```
- ```
     [not a blocker] We have a number of assembly version conflicts which may be solved by moving additional dependencies upstream. The creative component of this work is to find a way to embed interop types on some dependencies consumed via NuGet.
  ```
  
  So, I’d say this issue is complete, because we do indeed have a build system which gives us both VSIXes. Please advise based on your opinion of the above, and I’ll close this issue.
